### ISSUE TITLE
Issues in some pages

### ISSUE THREAD

**Issue by chack1172** [mybb/mybb2#218]
I installed mybb 2.0 repositories on xampp and I have some issues in some pages.
For example in user profile page I get this and another 2 errors:
ErrorException in profile_basic.twig line 6: An exception has been thrown during the rendering of a template ("Argument 1 passed to MyBB\Core\Database\Repositories\Eloquent\UserProfileFieldRepository::hasForProfileField() must be an instance of MyBB\Core\Database\Models\User, null given, called in C:\xampp\htdocs\mybb2\app\Presenters\ProfileField.php on line 135 and defined") in "user.profile_basic" at line 6.

I think variable "$group" in page UserController.php is empty because if I change it with "a" I get the page without errors.

Another errors are in new topic and new thread:
ErrorException in TopicRepository.php line 229: Trying to get property of non-object

Or posting registration:
QueryException in Connection.php line 669: SQLSTATE[42S22]: Column not found: 1054 Unknown column '0' in 'field list' (SQL: insert into `role_user` (`is_display`, `0`, `role_id`, `user_id`) values (1, 2, *items, 4))

Is it a xampp problem?

**Comment by euantorano** [comment]
Hi, these are known issues due to ongoing changes to the permissions and authentication system.

**Comment by chack1172** [comment]
Thanka for your reply, I thought it was a xampp error

**Comment by euantorano** [comment]
Yeah, I really need to finish the new permissions system. I'll make it a priority this week.
